EFS-CSI/README.md

```diff
@@ -14,6 +14,16 @@ EFS_ID           # EFS ID can be found from AWS Console or command-line:
 EFS_REGION       # EFS region
 EFS_SECURITY_GROUPS  # EFS seucurity group which is the subnet security group which should be same as EKS
 ```
+
+## Add NFS Inbound Rule
+Either from AWS Console or run following script to add NFS inbound rule if no such rule available yet.
```

Comment:
Either from the AWS Console or run the following script to add the NFS inbound rule if no such rule is present.

Reply:
fixed